### PR TITLE
Write README

### LINKED ISSUE #43: Need to write a readme
## Feature Request

**Is your feature request related to a problem? Please describe.**  

Currently, there is not enough explanation on how to use the library on GitHub and how to use the library.

**Describe the solution you'd like**  

I think we should explain it through a readme to make it easier for people to use.

**Describe alternatives you've considered**  

**Additional context**

### PR DESCRIPTION
**Pull request by HyeockJinKim**
# Summary 

- It closes #43 

## How to Review? 

- Please refer to my self-review submitted as the review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+npm install @0auth/client  // client library
+
+// using yarn
+yarn add @0auth/server     // server library
+yarn add @0auth/client     // client library
+```
+
+## Usage
+
+should be written later.
```
Comment (HyeockJinKim) [inline README.md:19]: When adding the client and server example codes, a simple code example will be added together.

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+
+## Example
+
+* example of [client](./examples/client)
+* example of [server](./examples/server)
```
Comment (HyeockJinKim) [inline README.md:24]: The example code has not been uploaded yet.

**Review by HyeockJinKim — COMMENTED** [review]
Self review

**Review by anthonyminyungi — APPROVED** [review]
Okay!

**Pull request merged**